api/app/controllers/spree/api/base_controller.rb

```diff
@@ -78,16 +81,29 @@ def authenticate_user
         end
       end
 
+      def load_user_roles
+        @current_user_roles = if @current_api_user
+          @current_api_user.spree_roles.pluck(:name)
+        else
+          []
+        end
+      end
+
       def unauthorized
-        render "spree/api/errors/unauthorized", :status => 401 and return
+        render "spree/api/errors/unauthorized", status: 401 and return
       end
 
-      def error_during_processing(exception)
+      def unprocessable_entity(exception)
         Rails.logger.error exception.message
         Rails.logger.error exception.backtrace.join("\n")
 
-        render :text => { :exception => exception.message }.to_json,
-          :status => 422 and return
+        render text: { exception: exception.message }.to_json,
+          status: 422 and return
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Use `&&` instead of `and`.